### PR TITLE
OBSDOCS-1294 Observability UI plugins overview

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.12+

Issue:
[OBSDOCS-1294](https://issues.redhat.com//browse/OBSDOCS-1294)

Link to docs preview:
https://81402--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+
+The {coo-first} can install and manage UI plugins to enhance the observability capabilities of the OpenShift web console. The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc]: This is not a product name, just a generic term

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project. With the troubleshooting panel available under *Observe* -> *Alerting* , you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
+The output of Korrel8r is displayed in the form of an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, pod, etc. 
+
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the OpenShift web console under *Observe* -> *Traces*. You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems. Select a supported Tempo insance running the cluster, and set a time range and query, to view the trace data.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc]: This is not a product name, just a generic term

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The {coo-first} can install and manage UI plugins to enhance the observability capabilities of the OpenShift web console.
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:13]: This is not a product name, just a generic term

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+With the troubleshooting panel available under *Observe* -> *Alerting* , you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
+The output of Korrel8r is displayed in the form of an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, pod, etc.
+
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the OpenShift web console under *Observe* -> *Traces*.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:29]: This is not a product name, just a generic term

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc]: Changed to "Select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+With the troubleshooting panel available under *Observe* -> *Alerting* , you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
+The output of Korrel8r is displayed in the form of an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, pod, etc.
+
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the {product-title} web console under *Observe* -> *Traces*.
+You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
+Select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster, and set a time range and query, to view the trace data.
+
+== CLuster logging
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:33]:
```suggestion
["observability-ui-plugins-overview-logging_{context}"]
== Cluster logging
```

Typo in heading, also all the H2 (`==`) headings in this assembly need to have anchor IDs added. I've provided an example. More info: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:9]: Suggest adding an attribute for the Operator name if there isn't one already

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The {coo-first} can install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:12]:
```suggestion
You can use the {coo-first} to install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
```
Suggest revising to make [active](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice) and avoid [anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism)

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
+
+== Dashboards
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus datasources from the cluster to the default dashboards, in addition to the in-cluster datasource.
+This results in a unified observability experience across different data sources.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:19]:
```suggestion
You can add other Prometheus data sources from the cluster to the default dashboards, in addition to the in-cluster data source.
This results in a unified observability experience across different data sources.
```

Both [IBMSG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#data-source) and [Prometheus docs](https://grafana.com/docs/grafana/latest/datasources/prometheus/) seem to use "data source".

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+With the troubleshooting panel available under *Observe* -> *Alerting* , you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:24]:
```suggestion
You can use the troubleshooting panel available from the *Observe* -> *Alerting* page to easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources across different data stores.
```

- Suggestions for active voice / minimalism and readability.
- Avoid use of "under" preposition for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements (not sure if **Alerting** is a page, menu, etc, so you may need to tweak this suggestion to fit)

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+The output of Korrel8r is displayed in the form of an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, pod, etc.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:25]:
```suggestion
The output of Korrel8r is displayed as an interactive node graph. When you click on a node, you are directed to the corresponding web console page with the specific information for that node.
```
Suggested editing for minimalism, I think the list of examples here might not be necessary. If including it, "etc" must not be used per https://www.ibm.com/docs/en/ibm-style?topic=word-usage#etc

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the {product-title} web console under *Observe* -> *Traces*.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:29]: - Mixed usage of "{product-title} web console" and just "web console" throughout the doc - I'd choose one and keep it consistent. Maybe use "{product-title} web console" for the first instance and just "web console" for all subsequent instances?
- Same comment as previous about avoiding use of "under" or other prepositions like this for accessibility

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
+Select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster, and set a time range and query, to view the trace data.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:31]:
```suggestion
You can select a supported `TempoStack` object or `TempoMonolithic` object running in the cluster and set a time range and query to view the trace data.
```
Make more active, also I don't think the commas here are necessary

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+== CLuster logging
+
+The logging UI plugin surfaces logging data in the {product-title} web console under *Observe* -> *Logs*.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:35]: - Mixed usage of "{product-title} web console" and just "web console" throughout the doc - I'd choose one and keep it consistent. Maybe use "{product-title} web console" for the first instance and just "web console" for all subsequent instances?
- Same comment as previous about avoiding use of "under" or other prepositions like this for accessibility

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the {product-title} web console under *Observe* -> *Traces*.
+You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
+Select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster, and set a time range and query, to view the trace data.
+
+== CLuster logging
+
+The logging UI plugin surfaces logging data in the {product-title} web console under *Observe* -> *Logs*. 
+You can specify filters, queries, time ranges and refresh rates, with the results displayed a list of collapsed logs, which can then be expanded to show more detailed information for each log.
```
Comment (abrennan89) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:36]:
```suggestion
You can specify filters, queries, time ranges, and refresh rates. Results are displayed a list of collapsed logs, which can then be expanded to show more detailed information for each log.
```
Suggested edits for readability

**Review by abrennan89 — COMMENTED** [review]
Some suggested comments and required updates. Nice work, @gabriel-rh !

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 6 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:9]: coo-full is there already - will use it here

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 7 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:12]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 6 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:9]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 5 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:33]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 8 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:19]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 9 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:24]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 10 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:25]: I will keep the list as it was added by an engineer but remove etc

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 11 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:29]: done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 12 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:31]: I think the instruction "Select" is pretty active so will leave it - also , eng have used instance explicitly here, so I will keep that. Removing commas done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 14 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:36]: Done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 13 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc]: Done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus data sources from the cluster to the default dashboards, in addition to the in-cluster data source.
+This results in a unified observability experience across different data sources.
+
+["troubleshooting_{context}"]
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+You can use the troubleshooting panel available from the *Observe* -> *Alerting* page to easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
```
Comment (jgbernalp) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:26]: @gabriel-rh The troubleshooting panel can be also triggered from the masthead application launcher, as seen in this PR: https://github.com/openshift/troubleshooting-panel-console-plugin/pull/67/files. This is enabled form 4.17. Would you think this is something we can add in this paragraph?

Comment (jgbernalp) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:26]: I think this is already included here: https://github.com/openshift/openshift-docs/pull/81463/files#diff-e8d2db0b08d6c6517eb6812f32c2a129172a3511808372eda637b1046931505bR13

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+The output of Korrel8r is displayed as an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, or pod.
+
+["distributed-tracing_{context}"]
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the web console on the *Observe* -> *Traces* page.
+You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
+You can select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster and set a time range and query to view the trace data.
```
Comment (jgbernalp) [inline observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:34]: We only support `TempoStack` with multi-tenancy enabled, other types like `TempoMonolithic` or `TempoStacks` without multi-tenancy are not shown

Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc]: Done

File: observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc (hunk 15 above)
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc:26]: Added here as well

**Pull request merged**